src/components/atoms/ButtonNG/ButtonNG.scss

```diff
@@ -9,11 +9,26 @@ $normal-button-width: 100%;
 // $normal-bg - the normal background color
 // $hover-bg  - the background color when button is :hover-ed
 $variants-enum: (
-  ("primary", $white, $primary--live, $primary--live-hover),
-  ("secondary", $white, opaque-white(0.2), opaque-white(0.25)),
-  ("white", $primary--live, $white, $white),
-  ("dark", $white, opaque-black(0.4), opaque-black(0.6)),
-  ("danger", $white, $input--invalid, lighten($input--invalid, 10%))
+  (
+    "primary",
+    $primary-content--over,
+    $primary-accent--under,
+    $primary--live-hover
```

Comment:
Why mixed?

Is it more likely 

```scss
    $primary--content-over,
    $primary--accent-under,
    $primary--live-hover
```
or
```scss
    $primary-content--over,
    $primary-accent--under,
    $primary-live--hover
```
?

Reply:
fixed in be7e3130da6a0e0b2b73a6e280423b17a7df9052